OutOfSchool/OutOfSchool.Common/GeoMathHelper.cs

```diff
@@ -6,6 +6,8 @@ public static class GeoMathHelper
     {
         public const int Resolution = 6;
 
+        public const int ResolutionForCity = 5;
+
         public const int KRingForResolution = 1;
 
         public const string ElasticRadius = "5000m";
```

Comment:
Comment is not related to this PR. Subsequent function `GetDistanceFromLatLonInKm()` - states (both in name and comment) that result is returned in kilometers, but I really doubt. Please fix it (if possible)